docs/stylesheets/extra.css

```
@@ -0,0 +1,309 @@
+/* ==========================================================================
+   celebrimbor — the visual identity of an Elven-smith's workshop.
+   Celebrimbor forged the Rings of Power in Eregion, the land of holly. The
+   palette is drawn from that world: holly-green and parchment by day, a
+   star-lit forge by night, with burnished gold (the rings, the forge-fire) and
+   mithril-silver as the two metals. Cinzel — a face cut in the manner of Roman
+   inscriptions — carries the headings like letters graven in a ring; EB
+   Garamond sets the body like a page from a tome.
+   ========================================================================== */
+
+@import url('https://fonts.googleapis.com/css2?family=Cinzel:wght@500;600;700&display=swap');
```

Stylelint (17.14.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Cinzel:wght@500;600;700&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Cinzel:wght@500;600;700&display=swap'" (import-notation)

(import-notation)